src/qt/openuridialog.cpp

```diff
@@ -32,6 +32,7 @@ QString OpenURIDialog::getURI()
 
 void OpenURIDialog::accept()
 {
+#ifdef ENABLE_WALLET
```

Comment:
Is this #ifdef necessary? IMO `openuridialog.cpp/.h` is only in `BITCOIN_QT_WALLET_CPP`.